platform-sdk/swirlds-common/src/main/java/com/swirlds/common/system/platformstatus/statusactions/StartedReplayingEventsAction.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.swirlds.common.system.platformstatus.statusactions;
+
+/**
+ * An action that is triggered when the platform starts replaying events from the preconsensus event stream.
+ */
+public record StartedReplayingEventsAction() implements PlatformStatusAction {}
```

Comment:
I originally had them in an enum, but split them out into records since doing so decreased the complexity of handling, and made it more clear what was going on.

The only actions without parameters are:
- `StartedReplayingEventsAction`
- `CatastrophicFailureAction`
- `FallenBehindAction`

These are rarely called, so there isn't a meaningful performance penalty